engine/contact/contact_test.go

```diff
@@ -0,0 +1,150 @@
+package contact_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/bitcoin-sv/go-paymail"
+	"github.com/bitcoin-sv/spv-wallet/engine"
+	"github.com/bitcoin-sv/spv-wallet/engine/contact/testabilities"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester/fixtures"
+)
+
+func Test_ClientService_AdminCreateContact_Success(t *testing.T) {
+	tests := map[string]struct {
+		contactPaymail   string
+		creatorPaymail   string
+		fullName         string
+		metadata         *engine.Metadata
+		expectedError    error
+		expectedStatus   engine.ContactStatus
```

Comment:
When created, we always expect the status to be `ContactNotConfirmed` so I believe that this field is not needed.